client/src/__mocks__/gatsby.ts

```diff
@@ -13,24 +14,14 @@ module.exports = {
   graphql: jest.fn(),
   Link: jest.fn().mockImplementation(
     // these props are invalid for an `a` tag
-    ({
-      activeClassName,
-      activeStyle,
-      getProps,
-      innerRef,
-      partiallyActive,
-      ref,
-      replace,
-      to,
```

Comment:
I believe these are all part of GatsbyLinkProps, so we should be discarding them.